torch/_dynamo/variables/misc.py

```diff
@@ -1460,7 +1460,14 @@ def call_method(
         if tx.export:
             # For export cases, we can just make debugging functions no-ops
             return
-        unimplemented("Logger not supported for non-export cases")
+        if config.disable_logs:
+            if name == "isEnabledFor":
+                return variables.ConstantVariable.create(False)
+            if name in ['info', 'debug', 'error', 'warning', 'critical', 'exception']:
+                return
```

Comment:
We should be returning `ConstantVariable.create(None)` here (and above as well).

Reply:
done

Reply:
I meant that the empty return statements `return` should be replaced with `return ConstantVariable.create(None)` (there is another instance of this above this highlighted code that should also be changed. `variables.ConstantVariable.create(False)` should remain unchanged (although this is technically incorrect behavior).